pgjdbc/src/main/java/org/postgresql/jdbc/EscapedFunctions.java

```diff
@@ -144,55 +163,50 @@ public static Method getFunction(String functionName) {
    * ceiling to ceil translation
    *
    * @param parsedArgs arguments
-   * @return sql call
    * @throws SQLException if something wrong happens
    */
-  public static String sqlceiling(List<?> parsedArgs) throws SQLException {
-    return singleArgumentFunctionCall("ceil(", "ceiling", parsedArgs);
+  public static void sqlceiling(StringBuilder buf, List<CharSequence> parsedArgs) throws SQLException {
```

Comment:
consider `List<? extends CharSequence>`. that gives callers a bit more flexibility and has no impact to the implementation in this class.